src/__tests__/spectral.jest.test.ts

```diff
@@ -72,4 +76,78 @@ describe('Spectral', () => {
       });
     });
   });
+
+  test('reports issues for correct files', async () => {
```

Comment:
Note, this test could land in CLI tests as we have 2-3 test cases covering `$refs` there.
I avoided doing it for now, as it would collide with "oclif to yargs" PR.
We can move it later if needed, but I find `spectral.jest.test.ts` a decent spot either.

Reply:
Throw a comment in there saying `// TODO could move to spectral.test.ts after #471` or something

Reply:
It's already in `spectral.jest.test.ts` :D
What I'm saying is that we have a few tests covering `$refs` placed in tests for cli lint command, therefore we could potentially have all of them in one place rather than spread across 2 files.